vaadin-button-flow-parent/vaadin-button-flow-integration-tests/src/test/java/com/vaadin/flow/component/button/tests/ButtonIT.java

```diff
@@ -276,6 +276,37 @@ public void removeDisabled_buttonWorksNormally() {
 
     }
 
+    @Test
+    public void disableOnClick_enableInSameRoundtrip_clientSideButtonIsEnabled() {
```

Comment:
This tests that the workaround for disabling and enabling in the same roundtrip still works. After just removing the workaround none of the ITs failed otherwise 🤷‍♂️